api/controllers/CampaignBotController.js

```diff
@@ -1,548 +1,454 @@
 "use strict";
 
+/**
+ * Imports.
+ */
 const logger = rootRequire('lib/logger');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const phoenix = rootRequire('lib/phoenix')();
 const helpers = rootRequire('lib/helpers');
-
 const connOps = rootRequire('config/connectionOperations');
 const dbRbSubmissions = require('../models/campaign/ReportbackSubmission')(connOps);
 const dbSignups = require('../models/campaign/Signup')(connOps);
 const dbUsers = require('../models/User')(connOps);
 
+/**
+ * Setup.
+ */
 const CMD_REPORTBACK = (process.env.GAMBIT_CMD_REPORTBACK || 'P');
 
 /**
- * CampaignBotController
- * @constructor
- * @param {integer} campaignId - our DS Campaign ID
+ * CampaignBotController.
  */
-function CampaignBotController(campaignId) {
-  this.campaignId = campaignId;
-  this.campaign = app.getConfig(app.ConfigName.CAMPAIGNS, campaignId);
-  if (!this.campaign) {
-    return;
-  }
-
-  // @todo Config variable for default CampaignBot ID
-  // @todo Store campaignbot property on our config.campaigns to override bot
-  // on a per DS Campaign basis.
-  this.bot = app.getConfig(app.ConfigName.CAMPAIGNBOTS, 41);
+class CampaignBotController {
 
-  var mobileCommonsCampaign = this.campaign.staging_mobilecommons_campaign;
+  constructor(req, res) {
```

Comment:
That's good to know for sure...  I've been following examples from [how the legacy code's been doing it](https://github.com/DoSomething/gambit/blob/1.0.0/app/lib/sms-games/index.js), I can def having less objects to build/destroy would be a plus because we need to do it for Current User, Campaign, Signup, Reportback in each CampaignBot request.  

Your code above https://github.com/DoSomething/gambit/pull/631#discussion_r80524106 looks very clean, I see what you mean about keeping `req` and `res` limited to `router.js` only ... or potentially [something like a policies directory.](https://github.com/DoSomething/quicksilver-api/blob/master/api/policies/sessionAuth.js)